packages/apollos-church-api/src/data/campuses/__tests__/__snapshots__/resolvers.tests.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Campus gets campuses 1`] = `
+Object {
+  "data": Object {
+    "campuses": null,
+  },
+  "errors": Array [
+    [GraphQLError: Cannot read property 'value' of undefined],
```

Comment:
I don't think this is what you want returned in your resolver.

Reply:
hence my comment above